skrub/_reporting/_data/templates/column-associations.html

```diff
@@ -8,6 +8,7 @@
                 <th scope="col">Column 1</th>
                 <th scope="col">Column 2</th>
                 <th scope="col"><a href="https://en.wikipedia.org/wiki/Cram%C3%A9r%27s_V">Cramér's V</a></th>
+                <th scope="col">Pearson's Correlation</a></th>
```

Comment:
```suggestion
                <th scope="col"><a href="https://en.wikipedia.org/wiki/Pearson_correlation_coefficient">Pearson's Correlation</a></th>
```

to be consistent with the other column we can add the link even though most people know what pearson correlation is. if you prefer not to add the link you need to remove the closing `</a>` tag, as there is no corresponding opening `<a>`